csuntechlab/iSTART
# STR-269 -- Fix notification for next module release date

# Description
Fixed functionality for notification, next module release date.
  
# Testing
1. Check for console errors relating to release date notifications.
2. Ensure the release date is the proper date.
3. Ensure release date is only shown when needed.
